<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="exp_slider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>115</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="exp_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exp_text">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>115</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="fps_text">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fps_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame rate</string>
   </property>
  </widget>
  <widget class="QSlider" name="fps_slider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="fps_u_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>145</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>fps</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="clk_combo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clk_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pixel clock</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="run_checkbox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="exp_u_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>115</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Camera control</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="aoi_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>265</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[0,1280,0,1024]</string>
   </property>
  </widget>
  <widget class="QLabel" name="aoi_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Area of interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="aoi_units">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>265</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>px</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="aoi_reset">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>290</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="aoi_set">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shutter_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Shutter mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="shutter_combo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>55</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="blmode_checkbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QLabel" name="bloffset_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bloffset_text">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="bloffset_slider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="autogain_checkbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gain_text">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="gain_slider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="gainboost_checkbox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>x2 boost</string>
   </property>
  </widget>
  <widget class="QLabel" name="gain_units">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>230</y>
     <width>10</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bit8_radio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8 bits</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">bitdepth_buttongroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="bit10_radio">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10 bits</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">bitdepth_buttongroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="bitdepth_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bit depth</string>
   </property>
  </widget>
  <widget class="QLabel" name="blacklevel_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Black level correction</string>
   </property>
  </widget>
  <widget class="QLabel" name="gain_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hardware gain</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="view_checkbox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="dark_checkbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dark correction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dark_view">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="dark_reset">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>exp_slider</zorder>
  <zorder>exp_label</zorder>
  <zorder>exp_text</zorder>
  <zorder>fps_text</zorder>
  <zorder>fps_label</zorder>
  <zorder>fps_u_label</zorder>
  <zorder>clk_combo</zorder>
  <zorder>clk_label</zorder>
  <zorder>exp_u_label</zorder>
  <zorder>title_label</zorder>
  <zorder>aoi_text</zorder>
  <zorder>aoi_label</zorder>
  <zorder>aoi_units</zorder>
  <zorder>aoi_reset</zorder>
  <zorder>aoi_set</zorder>
  <zorder>fps_slider</zorder>
  <zorder>shutter_label</zorder>
  <zorder>shutter_combo</zorder>
  <zorder>blmode_checkbox</zorder>
  <zorder>bloffset_label</zorder>
  <zorder>bloffset_text</zorder>
  <zorder>bloffset_slider</zorder>
  <zorder>autogain_checkbox</zorder>
  <zorder>gain_text</zorder>
  <zorder>gain_slider</zorder>
  <zorder>gainboost_checkbox</zorder>
  <zorder>gain_units</zorder>
  <zorder>bit8_radio</zorder>
  <zorder>bit10_radio</zorder>
  <zorder>bitdepth_label</zorder>
  <zorder>blacklevel_title</zorder>
  <zorder>gain_title</zorder>
  <zorder>view_checkbox</zorder>
  <zorder>run_checkbox</zorder>
  <zorder>dark_checkbox</zorder>
  <zorder>dark_view</zorder>
  <zorder>dark_reset</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bitdepth_buttongroup"/>
 </buttongroups>
</ui>
